crates/bitcoin-da/src/helpers/parsers.rs

```diff
@@ -69,6 +69,11 @@ pub trait VerifyParsed {
     fn signature(&self) -> &[u8];
     fn body(&self) -> &[u8];
 
+    /// Returns the hash of the body
+    fn get_unverified_hash(&self) -> Option<[u8; 32]> {
+        Some(calculate_sha256(self.body()))
+    }
+
     /// Verifies the signature of the inscription and returns the hash of the body
     fn get_sig_verified_hash(&self) -> Option<[u8; 32]> {
```

Comment:
A note to self irrelevant to this PR: wer are creating Secp256k1 context everytime from scratch in this method which is really really bad practice and quite expensive. Definitely change that.